packages/react-table/src/docs/demos/table-demos/StaticBottomPagination.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import {
+  Button,
+  Card,
+  Toolbar,
+  ToolbarContent,
+  ToolbarGroup,
+  ToolbarItem,
+  Pagination,
+  Select,
+  SelectVariant,
+  SelectOption,
+  PageSection,
+  Label
+} from '@patternfly/react-core';
+import { TableComposable, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+import { rows, columns } from '../../examples/Data.jsx';
+
+import FilterIcon from '@patternfly/react-icons/dist/esm/icons/filter-icon';
+import DashboardWrapper from '@patternfly/react-core/src/demos/examples/DashboardWrapper';
+
+export const StaticBottomPagination = () => {
+  const [isSelectOpen, setIsSelectOpen] = React.useState(false);
+  const [page, setPage] = React.useState(1);
+  const [perPage, setPerPage] = React.useState(10);
+  const [paginatedRows, setPaginatedRows] = React.useState(rows.slice(0, 10));
+  const handleSetPage = (_evt, newPage, perPage, startIdx, endIdx) => {
+    setPaginatedRows(rows.slice(startIdx, endIdx));
+    setPage(newPage);
+  };
+  handlePerPageSelect = (_evt, newPerPage, newPage, startIdx, endIdx) => {
+    setPaginatedRows(rows.slice(startIdx, endIdx));
+    setPage(newPage);
+    setPerPage(newPerPage);
+  };
+
+
+  const renderPagination = (variant, isCompact) => (
+    <Pagination
```

Comment:
This may be out of scope but I noticed a spacing difference between core and react, and looks like it's due to a difference in the toggle between the core and react demos. We recently made an update to the plain text options menu (what's used in pagination) that makes the entire toggle clickable versus just the toggle icon. @thatblindgeye added that in https://github.com/patternfly/patternfly-react/pull/7192/ as a `perPageComponent="button"` prop on `<Pagination>`, but our demos that reference `<Pagination>` are not using it.

I dunno if it makes sense to add that to this demo and have a separate issue to add it to the other demos - or handle it some other way, but wanted to mention it.